src/Date.php

```diff
@@ -29,7 +29,7 @@
  * - Uses either strptime() or DateTime to create a date from a format, depending on the format
  * - Uses strftime formatting for dates www.php.net/manual/en/function.strftime.php
  */
-class Date implements DateTimeInterface
```

Comment:
Hmmm, then it was bad from the first moment. @WanWizard might have been drunk. ;) There is an interface in resources for < PHP 5.5, it should be removed as well.